src/main/java/org/pesmypetcare/webservice/firebaseservice/adapters/firestore/FirestoreCollection.java

```diff
@@ -125,8 +130,8 @@ Query getDocumentsWhereArrayContains(@NonNull String collectionPath, @NonNull Fi
      * and the value should be equal to the specified value.
      *
      * @param collectionId Identifies the collections to query over. Every collection or subcollection with this ID as
-     *                     the last segment of its path will be included. Cannot contain a slash.
-     * @param field The name of the field to compare
+     *                     the last segment ofDocument its path will be included. Cannot contain a slash.
+     * @param field The name ofDocument the field to compare
```

Comment:
I guess you did replace all to 'of', fix this please

Reply:
Already done